compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -223,26 +223,51 @@ static LogicalResult setRootConfigForPackPipeline(func::FuncOp entryPointFn,
   return success();
 }
 
+/// Utility function that, given an element type, computes tiling scaling factor
+/// as : 64/bitWidth.
+/// Therefore, for example, if the element type has :-
+///     a. 64 bit width -> scaling factor will be 1.
+///     b. 32 bit width -> scaling factor will be 2.
+///     c. 16 bit width -> scaling factor will be 4.
+/// This can easily be scaled to other element data types by just changing the
+/// check in the `assert`.
+/// The rationale/need for this is - higher bit width element types can be
+/// accomodated using small tile sizes, whereas lower bit width element types
+/// can make use of larger tile sizes as they can be easily accomodated within
+/// the same. Therefore this tiling scale factor helps dynamically
+/// increase/decrease the tiling window depending on the element type's bit
+/// width.
+static unsigned getTilingScaleFactor(Type elemType) {
```

Comment:
It's better to move this function to Utils.cpp.

Reply:
Done.